pkg/apis/pipeline/v1beta1/pipelineref_validation.go

```diff
@@ -34,6 +35,7 @@ func (ref *PipelineRef) Validate(ctx context.Context) (errs *apis.FieldError) {
 
 	if ref.Resolver != "" || ref.Params != nil {
 		if ref.Resolver != "" {
+			errs = errs.Also(version.ValidateEnabledAPIFields(ctx, "resolver", config.BetaAPIFields).ViaField("resolver"))
```

Comment:
can this be added to `ValidateBetaFields`?

Reply:
I don't think that might be helpful for the codebase since that's pipelineRef rather than for the pipelineSpec?

Reply:
I see! Thanks!